### PR TITLE
2019 standards update

### PR DESCRIPTION
**Pull request by losborne**
Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-2134
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @hackrm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: spec/models/calculator_spec.js [hunk 1]
```diff
@@ -382,7 +385,7 @@ describe('Calculator', () => {
     });
   });
 
-  it('multiple observation measure correctly', () => {
+  xit('multiple observation measure correctly', () => {
```
Comment (hackrm) [inline spec/models/calculator_spec.js]: Should this skipped test have a TODO comment like the others?

Comment (losborne) [reply spec/models/calculator_spec.js]: Dont think it's necessary--It wouldn't hurt to have the comment, but adding commits on top of this PR for just a comment doesn't make sense IMO, as the code was reviewed in the PR into the feature branch
